src/ape/managers/project/manager.py

```diff
@@ -447,16 +447,20 @@ def _get_contract(self, name: str) -> Optional[ContractContainer]:
 
         return None
 
-    # @property
-    # def meta(self) -> PackageMeta:
-    #     return PackageMeta(**self.config_manager.get_config("ethpm").serialize())
+    @property
+    def meta(self) -> PackageMeta:
```

Comment:
- [ ] add doc str to this property

Reply:
docstring added: [e892b36](https://github.com/ApeWorX/ape/pull/802/commits/e892b367e0a8a711fa5a5f0e23d603044e96e4f8) please review for completeness

Reply:
added test, probably should be reworked: https://github.com/ApeWorX/ape/pull/802/commits/230b1b501ec196e81f94eebe3d3fcfec0735105a